zigpy/quirks/v2/__init__.py

```diff
@@ -243,15 +243,28 @@ class EntityMetadata:
     cluster_type: ClusterType = attrs.field(default=ClusterType.Server)
     initially_disabled: bool = attrs.field(default=False)
     attribute_initialized_from_cache: bool = attrs.field(default=True)
-    translation_key: str = attrs.field()
```

Comment:
It looks like `translation_key` was already "required" before, but there was never any validation to it.